api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -358,6 +360,17 @@ func (c *imageBuilder) createKanikoJobSpec(project mlp.Project, model *models.Mo
 							},
 						},
 					},
+					Tolerations: []v1.Toleration{
```

Comment:
We shouldn't hardcode it. Instead it should be configurarable the same way node selector does.